Shopify/shopify-cli-extensions
# Renamed next template folders

### WHY are these changes introduced?
Internal extension types names should be renamed as mention in this [proposal](https://docs.google.com/spreadsheets/d/1JDuPEYuZlPMVnIoCeSRrjbLLZER4ghbkvomSDQpBEUs/edit#gid=726475088&range=11:11) 

### WHAT is this pull request doing?
* Renamed every next extensions folder as they can be seeing in the proposal says.

### How to test your changes?
* Executing `make run create testdata/extension.config.yml` 
* Next extension folders in `tmp/` should have the correct names
